README.rst

```diff
@@ -29,11 +23,15 @@ The ``path`` can be omitted and defaults to the current directory.
 
 There are several different formats which can be used:
 
-* ``long`` Always include the git revision info.
-* ``short`` Never include the git revision info.
-* ``default`` Hide the git revision info unless the current release is a
-  pre-release or there are additional commits after the release.
-* ``json`` Output JSON data describing the version, in the form:
+* ``long`` Always include the git revision info.  For example,
+  ``1.0.0+git-1-123abcd`` means you are currently on commit ``123abcd``
+  which is one commit newer than the 1.0.0 release.
```

Comment:
Step in the right direction for sure. I get that this was a move, but I don't think this explanation really fits in this new format. It is useful information though, so I'm not sure what to do with it.

"for example, git tag x would translate to y for long, z for short, ..."

Maybe shell output for a few examples? Output for each of the tags: 1.0.0, 1.0.0rc1, and 1.0.0rc1+git-0-123abcd?